.gitignore

```diff
@@ -1,5 +1,7 @@
 **/*.svg
 **/*.png
+!**/testdata/*_golden.png
```

Comment:
do we want to ignore `_golden` files ?
I'd say we do want to see via `git status` that a `_golden` file was modified.
and/or see when a new `_golden` file appeared.